WeatherUI/Views/CityWeatherView/CityWeatherView.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  CityWeatherView.swift
+//  WeatherUI
+//
+//  Created by kostya on 04.07.2022.
+//
+
+import SwiftUI
+
+struct CityWeatherView: View {
+    let model: CityWeather
+
+    var body: some View {
+        ScrollView {
+            VStack(alignment: .leading, spacing: 0.0) {
+                HStack {
+                    Text(model.cityName)
+                        .font(.system(size: 18, weight: .regular))
```

Comment:
`regular` is the default value and could be omitted

Reply:
updated, also added custom font as per design